### ISSUE TITLE
Broken on 10.7.0?

### ISSUE THREAD

**Issue by jasnell** [nodejs/Gzemnid#22]
Seems to be broken on 10.7.0

```
james@ubuntu:~/node/Gzemnid$ node gzemnid.js meta
Reading meta directory...
Package list version: Mon Aug 13 2018 13:37:48 GMT-0700 (Pacific Daylight Time).
Reading: 50000...
Reading: 100000...
Reading: 150000...
Reading: 200000...
Reading: 250000...
Reading: 300000...
Reading: 350000...
Reading: 400000...
Reading: 450000...
Reading: 500000...
Reading: 550000...
Reading: 600000...
Reading: 650000...
Total packages: 677518.
To download: 677518.
Downloading: 0-0.0.0.json...
Downloading: 0--0.0.1.json...
Downloading: 0-_-0-1.0.0.json...
Downloading: 0-----1.0.0.json...
Downloading: 0-100-0.0.2.json...
Downloading: 0-9-0.0.1.json...
Downloading: 0.-1.0.0.json...
Downloading: 0.0-0.0.0.json...
Downloading: 0.0.1-1.0.1.json...
Downloading: 0.0.168-0.0.170.json...
RSS: 239.30078125 MiB
internal/buffer.js:35
    throw new ERR_OUT_OF_RANGE('value', `>= ${min} and <= ${max}`, value);
    ^

RangeError [ERR_OUT_OF_RANGE]: The value of "value" is out of range. It must be >= -2147483648 and <= 2147483647. Received 2215185706
    at checkInt (internal/buffer.js:35:11)
    at writeU_Int32LE (internal/buffer.js:515:3)
    at Buffer.writeInt32LE (internal/buffer.js:684:10)
    at Encoder._flush (/home/james/node/Gzemnid/node_modules/lz4/lib/encoder_stream.js:215:7)
    at Encoder.prefinish (_stream_transform.js:141:10)
    at Encoder.emit (events.js:182:13)
    at prefinish (_stream_writable.js:630:14)
    at finishMaybe (_stream_writable.js:638:5)
    at endWritable (_stream_writable.js:649:3)
    at Encoder.Writable.end (_stream_writable.js:589:5)
james@ubuntu:~/node/Gzemnid$
```

**Comment by ChALkeR** [comment]
@jasnell This is caused by node-lz4, see https://github.com/pierrec/node-lz4/issues/62 and my PR at https://github.com/pierrec/node-lz4/pull/65.

The work-around would be to either:
 1. Use Node.js 8.x LTS
 2. Disable compression in `config.local.json`
 3. Manually apply one-line patch from https://github.com/pierrec/node-lz4/pull/65 to the lz4 dependency in `node_modules`.

**Comment by ChALkeR** [comment]
lz4 PR landed in master, but it has not been released yet.

**Comment by ChALkeR** [comment]
@jasnell Should be fixed in a620ec9 now.
